beacon-chain/rpc/eth/beacon/pool.go

```diff
@@ -304,3 +305,23 @@ func (bs *Server) SubmitVoluntaryExit(ctx context.Context, req *ethpbv1.SignedVo
 
 	return &emptypb.Empty{}, nil
 }
+
+// ListBLSToExecutionChanges retrieves BLS to execution changes known by the node but not necessarily incorporated into any block
+func (bs *Server) ListBLSToExecutionChanges(ctx context.Context, _ *emptypb.Empty) (*v2.BLSToExecutionChangesPoolResponse, error) {
```

Comment:
Is there a reason why we are passing `_ *emptypb.Empty` if it's not being used?

Reply:
The signature must comply with an interface generated from this protobuf definition:
```
rpc ListBLSToExecutionChanges(google.protobuf.Empty) returns (v2.BLSToExecutionChangesPoolResponse)
```
If I am not mistaken, a protobuf function needs to have at least one parameter and `Empty` is a placeholder when you don't really need one.

Reply:
Thank you for clarifying